S3Proxy/cache.go

```diff
@@ -0,0 +1,53 @@
+package S3Proxy
+
+import "time"
+
+var s3Buckets = map[string]*s3BucketCacheItem{}
+var s3Objects = map[string]*s3ObjectCacheItem{}
+
+type s3BucketCacheItem struct {
+	Name      string
+	Location  string
+	Timestamp time.Time
+	TTL       time.Duration
+}
+
+func (b s3BucketCacheItem) String() string {
+	return "{Name:" + b.Name +
```

Comment:
out, _ := json.Marshal(bucket)
fmt.Println(string(out))

you should be able to modify the TTL in the out object. Or copy the original modify TTL and json.Marshal. So we don't need to update the return string if the struct definition changes.